prow/jobs/kyma/tests/end-to-end/upgrade-e2e/upgrade-e2e-tests-generic.yaml

```diff
@@ -0,0 +1,47 @@
+# Code generated by rendertemplates. DO NOT EDIT. 
+
+
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+job_template: &job_template
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
+  max_concurrency: 10
+  labels:
+    preset-dind-enabled: "true"
+    preset-sa-gcr-push: "true"
+    preset-docker-push-repository-kyma: "true"
+  extra_refs:
+    - <<: *test_infra_ref
+      base_ref: master
+  branches:
+    - ^master$
+    - ^release-(1\.13|1\.12|1\.11)-upgrade-e2e$
+  run_if_changed: "^tests/end-to-end/upgrade-e2e/|^common/makefiles/"
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/prow/test-infra/bootstrap:v20181204-a6e79be
+        securityContext:
+          privileged: true
+        command:
+          - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build-generic.sh"
+        args:
+          - "/home/prow/go/src/github.com/kyma-project/kyma/tests/end-to-end/upgrade-e2e"
+        resources:
+          requests:
+            memory: 1.5Gi
+            cpu: 0.8
+
+presubmits: # runs on PRs
+  kyma-project/kyma:
+    - name: pre-kyma-tests-end-to-end-upgrade-e2e
+      optional: false
```

Comment:
Why wouldn't you consider optional `true` at this moment as it includes changes for `common/makefiles/`?